charts/opentelemetry-collector/templates/_config.tpl

```diff
@@ -67,6 +67,7 @@ Build config file for deployment OpenTelemetry Collector
 {{- $values := deepCopy .Values }}
 {{- $data := dict "Values" $values | mustMergeOverwrite (deepCopy .) }}
 {{- $config := include "opentelemetry-collector.baseConfig" $data | fromYaml }}
+{{- $config = include "opentelemetry-collector.ballastConfig" $data | fromYaml | mustMergeOverwrite $config }}
```

Comment:
If we want include ballast in both deployment/statefulset and daemonset then I we can instead add it to `opentelemetry-collector.baseConfig`